2024-01-28 19:31:08,176 - Failed to launch 1_crypto_buy_ON.py: Command '['python', '1_crypto_buy_ON.py']' returned non-zero exit status 1.
2024-01-28 19:31:20,027 - 3_crypto_buy_ON_bt_update.py launched successfully.
2024-01-28 19:31:20,044 - Failed to launch  : Command '['python', ' ']' returned non-zero exit status 2.
